### PR TITLE
Abel Roy's Card

### PR DESCRIPTION
**Pull request by AbelR007**
Added a new card with helpful information on Web Development.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:117]: Please change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` 
into `<!-- ________ Your_Name Contributor card START ________ -->`

File: index.html [hunk 2]
```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
...
+                    Sololearn </a> - A great place to start learning programming.
+              </li>
+              <li>
+                  <a href="#" target="_blank" title="E Book">
+                    Book on "Learning Web Design" </a> [E Book, Available on Amazon]
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:143]: Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @AbelR007, thank you for contributing to this project. ❤
But there are a couple of small changes that need to be done before we go ahead and merge your code. Please change these two comments, because they are very helpful for everyone to identify each card. 
Once you've made those changes, please get back to us. We will be able to review and merge your pull request in a flash.

**Comment by AbelR007** [comment]
Thanks, @dilshankarunarathne  for responding,
I have corrected the card with the proper comments!

**Review by dilshankarunarathne — APPROVED** [review]

**Comment by dilshankarunarathne** [comment]
Congratulations 🎉❤
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Pull request merged**